src/content/reference/web-interface/_index.md

```diff
@@ -0,0 +1,31 @@
++++
+title = "Web Interface Reference"
+description = "Documentation on our web interface, a visual way to create and delete clusters, create key pairs and more."
+date = "2019-07-05"
+layout = "subsection"
+weight = 10
++++
+
+# Giant Swarm's Web Interface
+
+Hopefully, if we did our job well then our Web Interface is so intuitive and
+easy to use that this reference isn't necessary.
+
+However, for those of you interested in having an overview of what is possible
+with our Web Interface, and how to access it, read on!
+
+## What can I do with the Web Interface?
+
+Our Web Interface let's you do the following things. Some topics link to more
+detailed information:
+
+- View and manage clusters
+- View and manage keypairs for your clusters
+- View and manage organizations
+- Manage your own account and password
+- Learn how to get started with your kubernetes clusters by following a guide
+- [Add, Remove and configure Apps on your clusters.](app-catalog/)
+
+## Feedback {#feedback}
+
+We welcome your feedback our Web Interface. If you feel like sharing openly, use the GitHub repository and create an [issue](https://github.com/giantswarm/happa/issues), so other users can participate. Otherwise please use the common Giant Swarm support channels.
```

Comment:
```suggestion
We welcome your feedback to our web UI. If you feel like sharing openly, use the GitHub repository and create an [issue](https://github.com/giantswarm/happa/issues), so other users can participate. Otherwise please use the common Giant Swarm support channels.
```

Reply:
I think it should be 'feedback on' right? I made that change.

Reply:
`on` or `for` work in this context. `to` reads a bit strangely.